### DISCUSSION TITLE
Re-publish a page

### DISCUSSION THREAD

**Discussion by jamesking** [quarto-dev/quarto-cli#2849] [Q&A]
I recently published my first page to quarto.pub. Very easy, thank you!

I noticed an error in my published doc and wanted to correct it by re-publishing the document.

However, `quarto publish` does not let me overwrite the exiting publication and suggests a new publication slug, to avoid over-writing the exiting doc.

```
? Document name: › Use another name...
? Publish with name: › <existing-document-slug>
? The specified name is already in use within your account.
```

I worked around this by deleting the existing doc from quarto.pub and then re-publishing, but I think it would be more natural to offer a command-line option to overwrite doc. 

I am not using a `_publish.yml` to publish a whole site, just a single page document.

**Comment by kirahowe** [comment]
Is this possible to do? I often work iteratively and want to update the content at a given URL (overwriting it). The history of my work is available through git, so I don't feel a need to maintain multiple actual published versions. I'm wondering if I'm missing something obvious about how to publish something to an existing URL?
